features/box-command.feature

```diff
@@ -41,6 +41,9 @@ Feature: Command output from box command
     When I successfully run `bundle exec vagrant service-manager box ip`
     Then stdout from "bundle exec vagrant service-manager box ip" should contain "<ip>"
 
+    When I successfully run `bundle exec vagrant service-manager box ip --script-readable`
+    Then stdout from "bundle exec vagrant service-manager box ip --script-readable" should contain "IP=<ip>"
```

Comment:
there is also a _output_is_script_readable_ step (see support/env.rb) which ensures that the whole output is script readable. That is another additional important check. It verifies that all output lines re script readable